entity-framework/core/modeling/owned-entities.md

```diff
@@ -0,0 +1,81 @@
+---
+title: EF Core | Owned Entity Types | Microsoft Docs
+author: julielerman
+
+uid: core/modeling/owned-entities
+---
+# Owned Entity Types
+
+This feature was added to EF Core 2.0.
+
+Owned entity type allow you to map types that do not have their own identity and are used as properties in any of your entities (such as a value object). An owned entity type shares the same CLR type with another entity type. The entity containing the defining navigation is the owner. When querying the owner the owned types will be included by default.
+
+By convention a shadow primary key will be created for the owned type and it will be mapped to the same table as the owner by using table splitting. This allows to use owned types similarly to how complex types are used in EF6. 
+
+The OwnsOne fluent mapping alerts the DbContext to this mapping. In this example, StreetAddress is a type with no identity property. It is used as a property of the Order type to specify the shipping address for a particular order. In the DbContext, you can use the Entity().OwnsOne() method to specify that the ShippingAddress property is an Owned Entity of the Order type.
```

Comment:
>ShippingAddress property is an Owned Entity of the Order type.

This is a bit ambiguous, consider rewording as "ShippingAddress property points to an Owned Entity owned by the Order type"

Reply:
thanks @AndriySvyryd . I'm traveling for conferences and so quite distracted but will get to these fixes and a new PR when I get back home next week. :)

Reply:
I would prefer not capitalizing "owned entity". 

> The OwnsOne fluent mapping alerts the DbContext to this mapping

- Can we use this chance to mention that owned types are never discovered by convention in EF Core?
- Alert has strong warn connotation? Notify?
- EF Core instead of the DbContext?

> In the DbContext, you can use the Entity().OwnsOne() method...

In `OnModelCreating`?